WordPress/gutenberg
# Duotone filter not rendering in some places

## What?

Fixing duotone rendering in the pattern selection panel.

## Why?

This is necessary to make the pattern preview accurate.

## How?

With this change, the duotone presets will render if the block supports duotone and has no custom duotone applied.

Previously we were rendering just the custom duotone applied to the current block but if the block has a filter applied globally using `theme.json`, the duotone doesn't work because the preset filters are not rendered.

## Testing Instructions




1. Use a theme that supports duotone for example Skatepark
2. Create a new post or page
3. Open the add pattern panel
4. See the patterns that include blocks without a custom duotone applied but with duotone presets in `theme.json` for example "Full-width image with aside caption"  from Skatepark.

## Screenshots or screencast 

| **Before:**  | **After:** |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/1310626/168165194-51315521-5104-457d-8e36-3da2fb708708.png)  | ![image](https://user-images.githubusercontent.com/1310626/168164958-d3a358ba-d3c1-4f34-b39b-527b3fb6b990.png) |
| ![image](https://user-images.githubusercontent.com/1310626/168168751-8fe87359-9142-4de3-a2ab-5113a4114f64.png) | ![image](https://user-images.githubusercontent.com/1310626/168168785-0d9b6142-db51-4606-8c66-18668d6e7354.png) |
